docs/eample-findBy*.md

```diff
@@ -0,0 +1,142 @@
+---
+id: example-findBy*
+title: example `findByText`
+sidebar_label: findByText
+---
+
+```javascript
+// src/__tests__/example.test.js
+// This is an example of how to use findByText to query for text that
+// is not visible right away
+
+import {
+  getByText,
+  getByRole,
+  findByText,
+  getByPlaceholderText,
+} from '@testing-library/dom'
+import userEvent from '@testing-library/user-event'
+// provides a set of custom jest matchers that you can use to extend jest
+// i.e. `.toBeVisible`
+import '@testing-library/jest-dom'
+
+const renderContent = el => {
```

Comment:
This `renderContent` method is inspired by the example from the [example intro](https://testing-library.com/docs/dom-testing-library/example-intro)